.github/workflows/test-unit.yaml

```diff
@@ -0,0 +1,37 @@
+name: Unit Test - Python
```

Comment:
Please use this action to run your unit tests: https://github.com/kubeflow/sdk/blob/main/.github/workflows/test-python.yaml#L9


We should just run tests after we run `make verify` in the Github action.

Reply:
done